core-services/prow/02_config/generate-boskos.py

```diff
@@ -295,6 +296,9 @@
 for i in range(60,64):
     CONFIG['vsphere-discon-quota-slice']['qe-discon-segment-{}'.format(i)] = 1
 
+for i in range(230,235):
+    CONFIG['vsphere-dis-quota-slice']['devqe-segment-{}'.format(i)] = 1
```

Comment:
shall we add `discon` keyword in `devqe-segment` ? Or `devqe-segment` is the name configured in ibmc vsphere cluster, can not be changed.

Reply:
seems missing `-disconnected` here? should be `devqe-segment-{}-disconnected`.

Reply:
Good catch, I added it back, thanks.